tests/test_meshtastic_utils_async.py

```
@@ -0,0 +1,621 @@
+        Returns:
+            bool: `True`, indicating the event is considered set.
+        """
+        return True
+
+    def set(self) -> None:
+        """
+        Mark the event as set so subsequent is_set() calls return True.
+
+        Mimics threading.Event.set behavior for the test double.
+        """
+        return None
```

Ruff (0.15.12)

[warning] Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)